---
layout: single
permalink: /publication
title: 发表论文
---
† 同等贡献 *通讯作者

### 2020 
- Liu YL, Wang XY, Wei YY, Liu Z, Lu QW, Liu F, Zhang T\*, Peng RH\*. [Chromosome Painting Based on Bulked Oligonucleotides in Cotton](https://doi.org/10.3389/fpls.2020.00802) _**Frontiers in Plant Science**_ 2020, 11:802
- Braz G.T, Martins L. do, Zhang T, Albert P.S, Birchler J.A, Jiang JM\* [A universal chromosome identification system for maize and wild Zea species.](https://link.springer.com/article/10.1007%2Fs10577-020-09630-5) _**Chromosome Research**_ 2020, 28(2):183-194
- Zhao H.N†, Zhang W.L†\*, Zhang T†, Lin Y, Hu Y.D, Fang C, Jiang JM\*. [Genome-wide MNase hypersensitivity assay
unveils distinct classes of open chromatin
associated with H3K27me3 and DNA
methylation in _Arabidopsis thaliana_](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-1927-5) _**Genome Biology**_ 2020, 21:24
- Song X.Y, Song R.R, Zhou J.W, Yan W.K, Zhang T, Sun H.J, Xiao J, Wu Y.F, Xi M.L, Lou Q.F, Wang H.Y\*, Wang X.E\*. [Development and application of oligonucleotide-based chromosome painting for chromosome 4D of _Triticum aestivum L._](https://link.springer.com/article/10.1007/s10577-020-09627-0) _**Chromosome Research**_ 2020. [DOI:10.1007/s10577-020-09627-0](https://doi.org/10.1007/s10577-020-09627-0)
- Liu G.Q, Zhang Y\*, Zhang T\*. [Computational approaches for effective CRISPR guide RNA design and evaluation](https://www.sciencedirect.com/science/article/pii/S2001037019303551?via%3Dihub) _**Computational and Structural Biotechnology Journal**_ 2020, 18:35-44 
- Liu XY†, Sun S†, Wu Y, Zhou Y, Gu S.W, Yu H.X, Yi CD, Gu MH, Jiang JM, Liu B, Zhang T\*, Gong ZY\*. [Dual‐color oligo‐FISH can reveal chromosomal variations and evolution in Oryza species](https://onlinelibrary.wiley.com/doi/abs/10.1111/tpj.14522) _**The Plant Journal**_ 2020, 101:112-121
- Xin HY†, Zhang T†, Wu YF, Zhang WL, Zhang PD, Xi ML\*, Jiang JM. [An extraordinarily stable karyotype of the woody Populus species revealed by chromosome painting.](https://onlinelibrary.wiley.com/doi/epdf/10.1111/tpj.14536) _**The Plant Journal**_ 2020, 101:253-264
  - _Cover:_ [_**The Plant Journal**_ Volume 101, Issue 2](/pdf/2020/2020_The_Plant_Journal_cover.pdf) 


### 2019
- Liu S†, Liu G.Q†, Cheng P.F, Xue C, Zhou Y, Chen X, Ye L, Qiao Z.Y\*, Zhang T\*, Gong Z.Y\* [Genome-wide Profiling of Histone Lysine Butyrylation Reveals its Role in the Positive Regulation of Gene Transcription in Rice](https://thericejournal.springeropen.com/articles/10.1186/s12284-019-0342-6) _**Rice**_ 2019, 12:86 
- Alvarez J.M, Moyano T.C, Zhang T, Gras D.E, Herrea F.J, Araus V. O'Brien J.A, Carrillo L, Medina J, Vicente-Carbajosa J, Jiang JM, Gutierrez R.A\* [Local changes in chromatin accessibility and transcriptional networks underlying the nitrate response in Arabidopsis roots](https://www.sciencedirect.com/science/article/abs/pii/S1674205219302916) _**Molecular Plant**_ 2019, 12(12):1545-1560
- Ren QR, Zhong ZH, Wang Y, You Q, Li Q, Yuan MZ, He Y, Qi CY, Tang X, Zheng XL, Zhang T\*, Qi YP\*, Zhang Y\*. [Bidirectional promoter based CRISPR-Cas9 systems for
plant genome editing](https://www.frontiersin.org/articles/10.3389/fpls.2019.01173/) _**Frontiers in Plant Science**_ 2019, 10:1173
- Zhong ZH†, Sretenovic S†, Ren QR†, Yang LJ, Bao Y, Qi CY, Yuan MZ, He Y, Liu SS, Wang JH, Huang L, Wang Y, Baby DB, Wang D, Zhang T, Qi YP\*, Zhang Y\*. [Improving plant genome editing with high-fidelity xCas9 and non-canonical PAM-targeting Cas9-NG](https://www.cell.com/molecular-plant/fulltext/S1674-2052(19)30124-8) _**Molecular Plant**_ 2019, 12(7):1027-1036
- Albert P.S†, Zhang T†, Semrau K, Rouillard JM, Kao YH, Wang CJ, Danilova T.V, Jiang JM, and Birchler J\*. [Whole-chromosome paints in maize reveal rearrangements, nuclear domains, and chromosomal relationships](https://www.pnas.org/content/116/5/1679) _**Proceedings of the National Academy of Sciences of the United States of America**_ 2019, 16(5):1679-1685
- Malzahn AA†, Tang X†, Lee K, Ren QR, Sretenovic S, Zhang YX, Chen HQ, Kang MJ, Bao Y, Zheng XL, Deng KJ, Zhang T, Salcedo V, Wang K, Zhang Y\*, Qi YP\*. [Application of CRISPR-Cas12a temperature sensitivity for improved genome editing in rice, maize, and Arabidopsis.](https://rdcu.be/bjyiQ) _**BMC Biology**_ 2019, 17(1):9
- Tang X†, Ren QR†, Yang LJ, Bao Y, Zhong ZH, He Y, Liu SS, Qi CY, Liu BL, Wang Y, Sretenovic S, Zhang YX, Zheng XL, Zhang T\*, Qi YP\*, Zhang Y\*. [Single transcript unit CRISPR 2.0 systems for robust Cas9 and Cas12a mediated plant genome editing.](https://onlinelibrary.wiley.com/doi/abs/10.1111/pbi.13068) _**Plant Biotechnology Journal**_ 2019, 17(7):1431-1445

### 2018
- Wu ZG†, Fang DM†, Yang R†, Gao F, An XY, Zhou XX, Li YF, Yi CD, Zhang T, Liang CZ, Cui P\*, Cheng ZK\*, Luo Q\*. [De novo genome assembly of Oryza granulata reveals rapid genome expansion and adaptive evolution.](https://www.nature.com/articles/s42003-018-0089-4) _**Communications Biology**_ 2018, 1: 84
- Tang X†, Liu GQ†, Zhou JP†, Ren QR, You Q, Tian L, Xin XH, Zhong ZH, Liu BL, Zheng XL, Zhang DW, Malzahn A, Gong ZY, Qi YP\*, Zhang, T\*, and Zhang Y\*. (2018). [A large-scale whole-genome sequencing analysis reveals highly specific genome editing by both Cas9 and Cpf1(Cas12a) nucleases in rice.](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1458-5) _**Genome Biology**_ 2018, 19:84
- You Q, Zhong ZH, Ren QR, Hassan F, Zhang Y\*, Zhang T\*. [CRISPRMatch: An Automatic Calculation And Visualization Tool For High-throughput CRISPR Genome-editing Data Analysis.](http://www.ijbs.com/v14p0858.htm) _**International Journal of Biological Sciences**_ 2018, 14(8): 858-862.
- Yang XM†, Zhao HN†, Zhang T, Zeng ZX, Zhang PD, Zhu B, Han YH, Braz GT, Casler MD, Schmutz J, and Jiang JM\*. [Amplification and adaptation of centromeric repeats in polyploid switchgrass species.](https://nph.onlinelibrary.wiley.com/doi/10.1111/nph.15098) _**New Phytologist**_ 2018, 218:1645-1657
- Hou LL†, Xu M†, Zhang T†, Xu ZH, Wang WY, Zhang JX, Yu MM, Ji W, Zhu CW, Gong ZY, Gu MH, Jiang JM and Yu HX\*. [Chromosome painting and its applications in cultivated and wild rice.](https://rdcu.be/UyBT) _**BMC Plant Biology**_ 2018, 18: 110.
- Dong ZB†, Yu J†, Li H, Huang W, Xu L, Zhao Y, Zhang T, Xu W, Jiang JM, Su Z\*, and Jin WW\*. [Transcriptional and epigenetic adaptation of maize chromosomes in Oat-Maize addition lines.](https://doi.org/10.1093/nar/gky209) _**Nucleic Acids Research**_  2018, 46(10): 5012-5028.
- Zhong ZH†, Zhang YX†, You Q†, Tang X, Ren QR, Liu SS, Yang LJ, Wang Y, Liu XP, Liu BL, Zhang T, Zheng XL, Le Y, Zhang Y\*, and Qi YP\*. [Plant genome editing using FnCpf1 and LbCpf1 nucleases at redefined and altered PAM sites.](http://www.cell.com/molecular-plant/fulltext/S1674-2052(18)30094-7) _**Molecular Plant**_ 2018, 11: 999-1002.
- Xin H†, Zhang T†, Han YH, Wu YF, Shi JS, Xi ML\*, and Jiang JM. (2018)  [Chromosome painting and comparative physical mapping of the sex chromosomes in Populus tomentosa and Populus deltoides.](https://link.springer.com/article/10.1007/s00412-018-0664-y) _**Chromosoma**_  2018, 127(3): 313-321.
- Braz GT†, He L†, Zhao HN†, Zhang T†, Semrau K, Rouillard JM, Torres GA, Jiang JM\*. [Comparative oligo-FISH mapping: an efficient and powerful methodology to reveal karyotypic and chromosomal evolution.](http://www.genetics.org/content/208/2/513.full.pdf) _**Genetics**_ 2018, 208(2): 513-523.

### 2017
- Zhang R, Xue C, Liu GQ, Liu XY, Zhang ML, Wang X, Zhang T\*, and Gong ZY\*. Segmental Duplication of Chromosome 11 and its Implications for Cell Division and Genome-wide Expression in Rice. _**Scientific Reports**_ 2017, 1(7): 2689.
- Tang X†, Lowder LG†, Zhang T, Maizahn AA, Zheng XL, Voytas DF, Zhong ZH, Chen YY, Ren QR, Li Q, Kirkland ER, Zhang Y\*, and Qi YP\*. [A CRISPR-Cpf1 system for efficient genome editing and transcriptional repression in plants.](/pdf/2017/2017_Nature_Plants.pdf) _**Nature Plants**_ 2017, 3: 17018.
- Marand AP, Zhang T, Zhu B, and Jiang JM\*: [Towards genome-wide prediction and characterization of enhancers in plants.](/pdf/2016/BBE_2016_inpress.pdf) _**Biochimica et Biophysica Acta-Gene Regulatory Mechanisms**_ 2017, 1860(1): 131-139.
- Zhou JP†, Deng KJ†, Cheng Y, Zhong ZH, Tian L, Tang X, Tang AT, Zheng XL, Zhang T, Qi YP\*, and Zhang Y\*: CRISPR-Cas9 Based Genome Editing Reveals New Insights into MicroRNA Function and Regulation in Rice. _**Frontiers in Plant Science**_ 2017, 8:1598.

### 2016
- Zhang T, Marand A, Jiang JM\*: [PlantDHS: A Database for DNase I Hypertensive Sites in Plants.](/pdf/2016/NAR_2016.pdf) _**Nucleic Acids Research**_ 2016, 44(D1): D1148-D1153.

### 2015
- Zhang T†, Zhang WL†, and Jiang JM\*: [Genome-wide nucleosome occupancy and positioning and their impact on gene expression and evolution in plants.](/pdf/2015/Plant_Physiol_2015.pdf) _**Plant Physiology**_ 2015, 168(4): 1406-1416.
- Han YH†, Zhang T†, Thammapichai P, Weng YQ, and Jiang JM\*:  [Chromosome-specific painting in Cucumis species using bulked oligonucleotides.](/pdf/2015/Genetics_2015.pdf) _**Genetics**_ 2015, 200(3):771-779.
- Zhu B†, Zhang WL†, Zhang T†, Liu B, Jiang JM\*: [Genome-wide prediction and validation of intergenic enhancers in Arabidopsis thaliana using open chromatin signature.](/pdf/2015/Plant_Cell_2015.pdf) _**The Plant Cell**_ 2015, 27(9):2415-2426.

### 2014
- Zhang T, Li GR, Yang ZJ, Nevo E: Adaptive evolution of duplicated hsp17 genes in wild barley from microclimatically divergent sites of Israel. _**Genetics and Molecular Research**_ 2014, 13(1): 1220-1232.
- Zhang WL, Zhang T, Wu YF, Jiang JM: Open Chromatin in Plant Genomes. _**Cytogenetic and Genome Research**_ 2014(1-3), 143:18-27.
- Yang LM†, Koo, DH†, Li D, Zhang T, Jiang JM, Luan FS, Renner SS, Henaff E, Sanseverino W, Garcia-Mas J, Casacuberta J, Senalik DA, Simon PW, Chen JF, and Weng YQ: Next-generation sequencing, FISH mapping and synteny-based modeling reveal mechanisms of decreasing dysploidy in Cucumis. _**The Plant Journal**_ 2014, 77(1): 16-30.

### 2013
- Zhang T†, Talbert PB†, Zhang WL†, Wu YF, Yang ZJ, Henikoff JG, Henikoff S, Jiang JM: [The CentO satellite confers translational and rotational phasing on cenH3 nucleosomes in rice centromeres.](/pdf/2013/PNAS_2013.pdf) _**Proceedings of the National Academy of Sciences of the United States of America**_ 2013, 110(50): E4875–E4883.
  - _Commentary:_ Heslop-Harrison, J.S. and Schwarzacher, T. [Nucleosomes and centromeric DNA packaging.](/pdf/2013/PNAS_2013_comm.pdf)  _**Proceedings of the National Academy of Sciences of the United States of America**_ 2013, 110(50): 19974-19975.
- Iovene M, Zhang T, Lou Q, Buell CR, Jiang JM: Copy number variation in potato - an asexually propagated autotetraploid species. _**The Plant Journal**_ 2013, 75(1):80-89.
- Wei W, Zhang T, Lin D, Yang ZJ, Guo FB: Transcriptional abundance is not the single force driving the evolution of bacterial proteins. _**BMC Evolutionary Biology**_ 2013, 13(1):162.
- Yang ZJ, Zhang T, Lang T, Li G, Chen G, Nevo E: Transcriptome Comparative Profiling of Barley eibi1 Mutant Reveals Pleiotropic Effects of HvABCG31 Gene on Cuticle Biogenesis and Stress Responsive Pathways. _**International Journal of Molecular Sciences**_ 2013, 14(10):20478-20491.
- Dong ZB, Jiang C, Chen X, Zhang T, Ding L, Song W, Luo H, Lai J, Chen H, Liu R, Jin WW: Maize LAZY1 Mediates Shoot Gravitropism and Inflorescence Development through Regulating Auxin Transport, Auxin Signaling, and Light Response. _**Plant Physiology**_ 2013, 163(3):1306-1322.

### 2012
- Zhang WL†, Zhang T†, Wu YF†, Jiang JM: [Genome-Wide Identification of Regulatory DNA Elements and Protein-Binding Footprints Using Signatures of Open Chromatin in Arabidopsis.](/pdf/2012/Plant_Cell_2012.pdf) _**The Plant Cell**_ 2012, 24(7):2719-2731.

### 2011
- Yang ZJ, Zhang T, Li GR, Nevo E: Adaptive microclimatic evolution of the dehydrin 6 gene in wild barley at "Evolution Canyon", Israel. _**Genetica**_ 2011, 139(11-12):1429-1438.
- Wang XH, Zhang T, Wen ZN, Xiao H, Yang ZJ, Chen GX, Zhao X: The chromosome number, karyotype and genome size of the desert plant diploid Reaumuria soongorica (Pall.) Maxim. _**Plant Cell Reports**_ 2011, 30(6):955-964.
- Tang ZX, Fu SL, Ren ZL, Zhang T, Zou YT, Yang ZJ, Li GR, Zhou JP, Zhang HQ, Yan BJ, Zhang HY, Tan FQ: Diversity and evolution of four dispersed repetitive DNA sequences in the genus Secale. _**Genome**_ 2011, 54(4):285-300.

### 2010
- Li GR, Zhang T, Wei P, Jia J, Yang ZJ: Sequence analysis of alpha-gliadin genes from Aegilops tauschii native to China. _**Asian Journal of Agricultural Sciences**_ 2010, 2(4):128-135.
- Li GR, Zhang T, Ban Y, Yang ZJ: Molecular characterization and evolutionary analysis of alpha-gliadin genes from Eremopyrum bonaepartis (Triticeae). _**Journal of Agricultural Science (Toronto)**_ 2010, 2(4):30-36.

### 2009
- Yang ZJ, Zhang T, Bolshoy A, Beharav A, Nevo E: Adaptive microclimatic structural and expressional dehydrin 1 evolution in wild barley, Hordeum spontaneum, at 'Evolution Canyon', Mount Carmel, Israel. _**Molecular Ecology**_ 2009, 18(9):2063-2075.
- Yang ZJ, Li GR, Jia JQ, Zeng X, Lei MP, Zeng ZX, Zhang T, Ren ZL: Molecular cytogenetic characterization of wheat–Secale africanum amphiploids and derived introgression lines with stripe rust resistance. _**Euphytica**_ 2009, 167(2):197-202.
- Li GR, Liu C, Zeng ZX, Jia J-Q, Zhang T, Zhou JP, Ren ZL, Yang ZJ: Identification of α-gliadin genes in Dasypyrum in relation to evolution and breeding. _**Euphytica**_ 2009, 165(1):155-163.


### 2006
- Hu GK, Feng H, Zhang T, Yan Y, Wu B, Jiang Q, Wu JM, Zhang YZ: Molecular cloning of cDNAs for 14-3-3 and its protein interactions in a white-rot fungus Phanerochaete chrysosporium. _**Annals of Microbiology**_ 2006, 56(3):191-196.
