---
layout: article
title: Publication
comments: false
---

**† Equally Contributing Authors**

## 2024
* Xin HY, Wang YD, Zhang WL, Bao Y, Neumann P, Ning YH, Zhang T, Wu YF, Jiang N\*, Jiang JM\*, Xi ML\*. [Celine, a long interspersed nuclear element retrotransposon, colonizes in the centromeres of poplar chromosomes](https://academic.oup.com/plphys/advance-article/doi/10.1093/plphys/kiae214/7656883#). ***Plant Physiology*** DOI:  https://doi.org/10.1093/plphys/kiae214

* You HL, Tang D, Liu HX, Zhou Y, Li YF, Shen Y, Gong ZY, Yu HX, Gu MH, Jinag JM, Zhang T\*, Cheng ZK\*. [Chromosome ends initiate homologous chromosome pairing during rice meiosis](https://academic.oup.com/plphys/advance-article/doi/10.1093/plphys/kiae152/7628145?login=true). ***Plant Physiology*** DOI: https://doi.org/10.1093/plphys/kiae152

* Han YS†, Liu GQ†, Wu YC, Bao Y, Zhang Y\*, Zhang T\*. [CrisprStitch: Fast evaluation of the efficiency of CRISPR editing systems](https://www.cell.com/plant-communications/fulltext/S2590-3462(23)00341-3). ***Plant Communications*** 2024, 5(3):100783 

* Chen L, Liu GQ, Zhang T\*. [Integrating machine learning and genome editing for crop improvement](https://link.springer.com/article/10.1007/s42994-023-00133-5). ***aBIOTECH***  DOI: https://doi.org/10.1007/s42994-023-00133-5 

## 2023

* Wang ZY, Lei YJ, Liu GQ, Ning Y, Ning YR, Zhang T\*, Xi M\*. [Male-Specific Sequence in Populus simonii Provides Insights into Gender Determination of Poplar](https://doi.org/10.3390/f14122385). ***Forests*** 2023, 14:2385

* Gurel F†, Wu YC†, Pan CT, Cheng YH,Li G ,Zhang T\*, Qi YP\*. [On- and Off-Target Analyses of CRISPR-Cas12b Genome Editing Systems in Rice](https://www.liebertpub.com/doi/10.1089/crispr.2022.0072). ***The CRISPR Journal***  2023, 6:62-74

* Zhong ZH†, Liu GQ†, Tang ZJ†, Xiang SH†, Yang L, Huang L, He Y, He Y, Fan TT, Liu SS, Zheng XL, Zhang T, Qi YP\*, Huang J\*, Zhang Y\*. [Efficient plant genome engineering using a probiotic sourced CRISPR-Cas9 system](https://www.nature.com/articles/s41467-023-41802-9#citeas). ***Nature Communications*** 2023, 14:6102

* Zhang YX†, Wu YC†, Li G†, Qi A, Zhang Y, Zhang T\*, Qi Y\*. [Genome-wide investigation of multiplexed CRISPR-Cas12a-mediated editing in rice](https://acsess.onlinelibrary.wiley.com/doi/10.1002/tpg2.20266). ***Plant Genome*** 2023, 16:e20266

* Sretenovic S†, Green Y†, Wu YC†, Cheng YH†, Zhang T\*, Eck J.V\*, Qi YP\*.[Genome- and transcriptome-wide off-target analyses of a high-efficiency adenine base editor in tomato](https://academic.oup.com/plphys/article-abstract/193/1/291/7198002?redirectedFrom=fulltext&utm_source=etoc&utm_campaign=plphys&utm_medium=email). ***Plant Physiology*** 2023, 193:291–303

* Chen C†, Han YS†, Xiao H, Zou BC, Wu DD, Sha LN, Yang CR, Liu SQ, Cheng YR, Wang Y, Kang HY, Fan X, Zhou YH, Zhang T\* , Zhang HQ\*. [Chromosome-specific painting in Thinopyrum species using bulked oligonucleotides](https://link.springer.com/article/10.1007/s00122-023-04423-w). ***Theoretical and Applied Genetics*** 2023, 136:177
  
* Bao Y†, Wei YY†, Liu YL†, Gao JJ, Cheng S, Liu GQ, You Q, Liu P, Lu QW, Li PT, Zhang SL, Hu N, Han YS, Liu S, Wu YC, Yang QQ, Li ZG, Ao GW, Liu F, Wang KB, Jiang JM, Zhang T\*, Zhang WL\* ,Peng RH\*. [Genome-wide chromatin accessibility landscape and dynamics of transcription factor networks during ovule and fiber development in cotton](https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-023-01665-4). ***BMC Biology*** 2023, 21:165

* Zhou JP†, Liu GQ†, Zhao YX, Zhang R, Tang X, Li L, Jia XY, Guo YC, Han YS, Bao Y, He Y, Han QQ, Yang H, Zheng XL, Qi YP\*, Zhang T\*, Zhang Y\*. [An efficient CRISPR–Cas12a promoter editing system for crop improvement](http://zhangtaolab.org/pdf/2023/2023_Nature_Plants.pdf). ***Nature Plants*** 2023, 9:588-604

## 2022

* Liu S†, Bao Y†, Deng H†, Liu GQ, Han YS, Wu YC, Zhang T\*, Chen C\*. [The Methylation Inhibitor 5-Aza-2′-Deoxycytidine Induces Genome-Wide Hypomethylation in Rice](https://thericejournal.springeropen.com/articles/10.1186/s12284-022-00580-6) ***Rice*** 2022, 15:35

* Xue C†, Liu GQ†, Sun S, Liu XY, Guo R, Cheng ZK, Yu HX, Gu MH, Liu K, Zhou Y\*, Zhang T\*, Gong ZY\*. [*De Novo* Centromere Formation in Pericentromeric Region of Rice Chromosome 8.](https://onlinelibrary.wiley.com/doi/epdf/10.1111/tpj.15862) ***The Plant Journal*** 2022, 111(3):859-871

* Wu YC†, Ren QR†, Zhong ZH†, Liu GQ†, Han YS, Bao Y, Liu L, Xiang SY, Liu S, Tang X, Zhou JP, Zheng XL, Sretenovic S, Zhang T\*, Qi YP\*, Zhang Y\*. [Genome-wide analyses of PAM-relaxed Cas9 genome editors reveal substantial off-target effects by ABE8e in rice.](https://onlinelibrary.wiley.com/doi/10.1111/pbi.13838) ***Plant Biotechnology Journal*** 2022, 20(9): 1670-1682

* Tao ST†, Liu P†, Shi YN, Feng YL, Gao JJ, Chen LF, Zhang AC, Cheng XJ, Wei HR, Zhang T\*, Zhang WL\*. [Single-Cell Transcriptome and Network Analyses Unveil Key Transcription Factors Regulating Mesophyll Cell Development in Maize.](https://www.mdpi.com/2073-4425/13/2/374/htm) ***Genes*** 2022, 13(2):374

* Feng YL, Tao ST, Zhang PY, Sperti FR, Liu GQ, Cheng XJ, Zhang T, Yu HX, Wang XE, Cheng CY, Monchaud D, Zhang WL\*. [Epigenomic features of DNA G-quadruplexes and their roles in regulating rice gene transcription.](https://academic.oup.com/plphys/advance-article/doi/10.1093/plphys/kiab566/6454112) ***Plant Physiology*** 2022, 188(3): 1632–1648

* Wu YC†, He Y†, Sretenovic S, Liu SS, Cheng YH, Han YS, Liu GQ, Bao Y, Fang Q, Zheng XL, Zhou JP, Qi YP\*, Zhang Y\*, Zhang T\*. [CRISPR-BETS: A base editing design tool for generating stop codons.](https://onlinelibrary.wiley.com/doi/abs/10.1111/pbi.13732) ***Plant Biotechnology Journal*** 2022, 20(3):499-510

## 2021

* Ding Y, Zhu JN, Zhao DS, Liu QQ, Yang QQ\*, ZhangT\*. [Targeting *Cis*-Regulatory Elements for Rice Grain Quality Improvement.](https://www.frontiersin.org/articles/10.3389/fpls.2021.705834/full) ***Frontiers in Plant Science*** 2021, 12:705834

* Liu GQ, Zhang T\*. [Single Copy Oligonucleotide Fluorescence In Situ Hybridization Probe Design Platforms: Development, Application and Evaluation.](https://www.mdpi.com/1422-0067/22/13/7124) ***International Journal of Molecular Sciences*** 2021, 22(13):7124

* Randall L.B†, Sretenovic S†, Wu YC†, Yin D, Zhang T, Eck J.V, Qi YP\*. [Genome- and transcriptome-wide off-target analyses of an improved cytosine base editor.](https://academic.oup.com/plphys/advance-article-abstract/doi/10.1093/plphys/kiab264/6294312) ***Plant Physiology*** 2021, 187(1):73-87

* Ren QR†, Sretenovic S†, Liu GQ†, Zhong ZH, Wang JH, Huang L, Tang X, Guo YC, Liu L, Wu YC, Zhou J, Zhao YX, Yang H, He Y, Liu SS, Yin DS, Mayorga R, Zheng XL, Zhang T\*, Qi YP\*, Zhang Y\*. [Improved plant cytosine base editors with high editing activity, purity, and specificity.](https://onlinelibrary.wiley.com/doi/abs/10.1111/pbi.13635) ***Plant Biotechnology Journal*** 2021, 19(10):2052-2068

* Zhang T†\*, Liu GQ†, Zhao HN, Braz G.T, Jiang JM\*. [Chorus2: design of genome‐scale oligonucleotide‐based probes for fluorescence in situ hybridization.](https://onlinelibrary.wiley.com/doi/abs/10.1111/pbi.13610) ***Plant Biotechnology Journal*** 2021, 19(10):1967-1978

* Meng FL†, Zhao HN†, Zhu B, Zhang T, Yang, Li Y, Han YP, Jiang JM\*. [Genomic Editing of Intronic Enhancers Unveils Their Role in Fine-Tuning Tissue-Specific Gene Expression in *Arabidopsis thaliana*.](https://academic.oup.com/plcell/advance-article/doi/10.1093/plcell/koab093/6187954) ***The Plant Cell*** 2021, 33(6):1997-2014

* Zhang C, Zhang JX, Tang YJ, Liu KW, Liu Y, Tang JQ, Zhang T, Yu HX\*. [DEEP GREEN PANICLE1 suppresses GOLDEN2-LIKE activity to reduce chlorophyll synthesis in rice glumes.](https://academic.oup.com/plphys/article/185/2/469/6020270) ***Plant Physiology*** 2021, 185(2):469-477

* Liu GQ, Qi YP, Zhang T. [Analysis of Off-Target Mutations in CRISPR-Edited Rice Plants Using Whole-Genome Sequencing.](https://link.springer.com/protocol/10.1007/978-1-0716-1068-8_10) ***Methods in Molecular Biology*** 2021, 2238:145-172

* Li GR†, Zhang T†, Yu ZH, Wang HJ, Yang EN, Yang ZJ\*. [An efficient Oligo‐FISH painting system for revealing chromosome rearrangements and polyploidization in Triticeae.](https://onlinelibrary.wiley.com/doi/10.1111/tpj.15081) ***The Plant Journal*** 2021, 105(4):978-993

## 2020

* Liu YL, Wang XY, Wei YY, Liu Z, Lu QW, Liu F, Zhang T\*, Peng RH\*. [Chromosome Painting Based on Bulked Oligonucleotides in Cotton.](https://doi.org/10.3389/fpls.2020.00802) ***Frontiers in Plant Science*** 2020, 11:802

* Zhao HN†, Zhang WL†\*, Zhang T†, Lin Y, Hu YD, Fang C, Jiang JM\*. [Genome-wide MNase hypersensitivity assay unveils distinct classes of open chromatin associated with H3K27me3 and DNA methylation in *Arabidopsis thaliana*.](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-1927-5) ***Genome Biology*** 2020, 21:24

* Song XY, Song RR, Zhou JW, Yan WK, Zhang T, Sun HJ, Xiao J, Wu YF, Xi ML, Lou QF, Wang HY\*, Wang XE\*. [Development and application of oligonucleotide-based chromosome painting for chromosome 4D of *Triticum aestivum* L.](https://link.springer.com/article/10.1007/s10577-020-09627-0) ***Chromosome Research*** 2020, 8(2):177-182

* Braz G.T, Martins L. do, Zhang T, Albert P.S, Birchler J.A, Jiang JM\* [A universal chromosome identification system for maize and wild Zea species.](https://link.springer.com/article/10.1007%2Fs10577-020-09630-5) ***Chromosome Research*** 2020, 28(2):183-194

* Liu GQ, Zhang Y\*, Zhang T\*. [Computational approaches for effective CRISPR guide RNA design and evaluation.](https://www.sciencedirect.com/science/article/pii/S2001037019303551?via%3Dihub) ***Computational and Structural Biotechnology Journal*** 2020, 18:35-44

* Liu XY†, Sun S†, Wu Y, Zhou Y, Gu SW, Yu HX, Yi CD, Gu MH, Jiang JM, Liu B, Zhang T\*, Gong ZY\*. [Dual‐color oligo‐FISH can reveal chromosomal variations and evolution in Oryza species.](https://onlinelibrary.wiley.com/doi/abs/10.1111/tpj.14522) ***The Plant Journal*** 2020, 101:112-121

* Xin HY†, Zhang T†, Wu YF, Zhang WL, Zhang PD, Xi ML\*, Jiang JM. [An extraordinarily stable karyotype of the woody Populus species revealed by chromosome painting.](https://onlinelibrary.wiley.com/doi/epdf/10.1111/tpj.14536) ***The Plant Journal*** 2020, 101:253-264

  * Cover: [The Plant Journal Volume 101, Issue 2](http://zhangtaolab.org/pdf/2020/2020_The_Plant_Journal_cover.pdf)

## 2019

* Liu S†, Liu GQ†, Cheng PF, Xue C, Zhou Y, Chen X, Ye L, Qiao ZY\*, Zhang T\*, Gong Z.Y\* [Genome-wide Profiling of Histone Lysine Butyrylation Reveals its Role in the Positive Regulation of Gene Transcription in Rice.](https://thericejournal.springeropen.com/articles/10.1186/s12284-019-0342-6) ***Rice*** 2019, 12:86

* Alvarez J.M, Moyano T.C, Zhang T, Gras D.E, Herrea F.J, Araus V. O’Brien J.A, Carrillo L, Medina J, Vicente-Carbajosa J, Jiang JM, Gutierrez R.A\* [Local changes in chromatin accessibility and transcriptional networks underlying the nitrate response in *Arabidopsis* roots.](https://www.sciencedirect.com/science/article/abs/pii/S1674205219302916) ***Molecular Plant*** 2019, 12(12):1545-1560

* Ren QR, Zhong ZH, Wang Y, You Q, Li Q, Yuan MZ, He Y, Qi CY, Tang X, Zheng XL, Zhang T\*, Qi YP\*, Zhang Y\*. [Bidirectional promoter based CRISPR-Cas9 systems for plant genome editing](https://www.frontiersin.org/articles/10.3389/fpls.2019.01173/) ***Frontiers in Plant Science*** 2019, 10:1173

* Zhong ZH†, Sretenovic S†, Ren QR†, Yang LJ, Bao Y, Qi CY, Yuan MZ, He Y, Liu SS, Wang JH, Huang L, Wang Y, Baby DB, Wang D, Zhang T, Qi YP\*, Zhang Y\*. [Improving plant genome editing with high-fidelity xCas9 and non-canonical PAM-targeting Cas9-NG.](https://www.cell.com/molecular-plant/fulltext/S1674-2052(19)30124-8) ***Molecular Plant*** 2019, 12(7):1027-1036

* Albert P.S†, Zhang T†, Semrau K, Rouillard JM, Kao YH, Wang CJ, Danilova T.V, Jiang JM, and Birchler J\*. [Whole-chromosome paints in maize reveal rearrangements, nuclear domains, and chromosomal relationships.](https://www.pnas.org/content/116/5/1679) ***Proceedings of the National Academy of Sciences of the United States of America*** 2019, 16(5):1679-1685

* Malzahn AA†, Tang X†, Lee K, Ren QR, Sretenovic S, Zhang YX, Chen HQ, Kang MJ, Bao Y, Zheng XL, Deng KJ, Zhang T, Salcedo V, Wang K, Zhang Y\*, Qi YP\*. [Application of CRISPR-Cas12a temperature sensitivity for improved genome editing in rice, maize, and *Arabidopsis*.](https://rdcu.be/bjyiQ) ***BMC Biology*** 2019, 17(1):9

* Tang X†, Ren QR†, Yang LJ, Bao Y, Zhong ZH, He Y, Liu SS, Qi CY, Liu BL, Wang Y, Sretenovic S, Zhang YX, Zheng XL, Zhang T\*, Qi YP\*, Zhang Y\*. [Single transcript unit CRISPR 2.0 systems for robust Cas9 and Cas12a mediated plant genome editing.](https://onlinelibrary.wiley.com/doi/abs/10.1111/pbi.13068) ***Plant Biotechnology Journal*** 2019, 17(7):1431-1445

## 2018

* Wu ZG†, Fang DM†, Yang R†, Gao F, An XY, Zhou XX, Li YF, Yi CD, Zhang T, Liang CZ, Cui P\*, Cheng ZK\*, Luo Q\*. [De novo genome assembly of *Oryza granulata* reveals rapid genome expansion and adaptive evolution.](https://www.nature.com/articles/s42003-018-0089-4) ***Communications Biology*** 2018, 1:84

* Tang X†, Liu GQ†, Zhou JP†, Ren QR, You Q, Tian L, Xin XH, Zhong ZH, Liu BL, Zheng XL, Zhang DW, Malzahn A, Gong ZY, Qi YP\*, Zhang, T\*, and Zhang Y\*. (2018). [A large-scale whole-genome sequencing analysis reveals highly specific genome editing by both Cas9 and Cpf1(Cas12a) nucleases in rice.](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1458-5) ***Genome Biology*** 2018, 19:84

* You Q, Zhong ZH, Ren QR, Hassan F, Zhang Y\*, Zhang T\*. [CRISPRMatch: An Automatic Calculation And Visualization Tool For High-throughput CRISPR Genome-editing Data Analysis.](http://www.ijbs.com/v14p0858.htm) ***International Journal of Biological Sciences*** 2018, 14(8):858-862.

* Yang XM†, Zhao HN†, Zhang T, Zeng ZX, Zhang PD, Zhu B, Han YH, Braz GT, Casler MD, Schmutz J, and Jiang JM\*. [Amplification and adaptation of centromeric repeats in polyploid switchgrass species.](https://nph.onlinelibrary.wiley.com/doi/10.1111/nph.15098) ***New Phytologist*** 2018, 218:1645-1657

* Hou LL†, Xu M†, Zhang T†, Xu ZH, Wang WY, Zhang JX, Yu MM, Ji W, Zhu CW, Gong ZY, Gu MH, Jiang JM and Yu HX\*. [Chromosome painting and its applications in cultivated and wild rice.](https://rdcu.be/UyBT) ***BMC Plant Biology*** 2018, 18:110.

* Dong ZB†, Yu J†, Li H, Huang W, Xu L, Zhao Y, Zhang T, Xu W, Jiang JM, Su Z\*, and Jin WW\*. [Transcriptional and epigenetic adaptation of maize chromosomes in Oat-Maize addition lines.](https://doi.org/10.1093/nar/gky209) ***Nucleic Acids Research*** 2018, 46(10):5012-5028.

* Zhong ZH†, Zhang YX†, You Q†, Tang X, Ren QR, Liu SS, Yang LJ, Wang Y, Liu XP, Liu BL, Zhang T, Zheng XL, Le Y, Zhang Y\*, and Qi YP\*. [Plant genome editing using FnCpf1 and LbCpf1 nucleases at redefined and altered PAM sites.](http://www.cell.com/molecular-plant/fulltext/S1674-2052(18)30094-7) ***Molecular Plant*** 2018, 11:999-1002.

* Xin H†, Zhang T†, Han YH, Wu YF, Shi JS, Xi ML\*, and Jiang JM. [Chromosome painting and comparative physical mapping of the sex chromosomes in *Populus tomentosa* and *Populus deltoides*.](https://link.springer.com/article/10.1007/s00412-018-0664-y) ***Chromosoma*** 2018, 127(3):313-321.

* Braz GT†, He L†, Zhao HN†, Zhang T†, Semrau K, Rouillard JM, Torres GA, Jiang JM\*. [Comparative oligo-FISH mapping: an efficient and powerful methodology to reveal karyotypic and chromosomal evolution.](http://www.genetics.org/content/208/2/513.full.pdf) ***Genetics*** 2018, 208(2):513-523.

## 2017

* Zhang R, Xue C, Liu GQ, Liu XY, Zhang ML, Wang X, Zhang T\*, and Gong ZY\*. Segmental Duplication of Chromosome 11 and its Implications for Cell Division and Genome-wide Expression in Rice. ***Scientific Reports*** 2017, 1(7):2689.

* Tang X†, Lowder LG†, Zhang T, Maizahn AA, Zheng XL, Voytas DF, Zhong ZH, Chen YY, Ren QR, Li Q, Kirkland ER, Zhang Y\*, and Qi YP\*. [A CRISPR-Cpf1 system for efficient genome editing and transcriptional repression in plants.](http://zhangtaolab.org/pdf/2017/2017_Nature_Plants.pdf) ***Nature Plants*** 2017, 3:17018.

* Marand AP, Zhang T, Zhu B, and Jiang JM\*. [Towards genome-wide prediction and characterization of enhancers in plants.](http://zhangtaolab.org/pdf/2016/BBE_2016_inpress.pdf) ***Biochimica et Biophysica Acta-Gene Regulatory Mechanisms*** 2017, 1860(1):131-139.

* Zhou JP†, Deng KJ†, Cheng Y, Zhong ZH, Tian L, Tang X, Tang AT, Zheng XL, Zhang T, Qi YP\*, and Zhang Y\*. CRISPR-Cas9 Based Genome Editing Reveals New Insights into MicroRNA Function and Regulation in Rice. ***Frontiers in Plant Science*** 2017, 8:1598.

## 2016

* Zhang T, Marand A, Jiang JM\*. [PlantDHS: A Database for DNase I Hypertensive Sites in Plants.](http://zhangtaolab.org/pdf/2016/NAR_2016.pdf) ***Nucleic Acids Research*** 2016, 44(D1):D1148-D1153.

## 2015

* Zhang T†, Zhang WL†, and Jiang JM\*. [Genome-wide nucleosome occupancy and positioning and their impact on gene expression and evolution in plants.](http://zhangtaolab.org/pdf/2015/Plant_Physiol_2015.pdf) ***Plant Physiology*** 2015, 168(4):1406-1416.

* Han YH†, Zhang T†, Thammapichai P, Weng YQ, and Jiang JM\*. [Chromosome-specific painting in *Cucumis* species using bulked oligonucleotides.](http://zhangtaolab.org/pdf/2015/Genetics_2015.pdf) ***Genetics*** 2015, 200(3):771-779.

* Zhu B†, Zhang WL†, Zhang T†, Liu B, Jiang JM\*. [Genome-wide prediction and validation of intergenic enhancers in Arabidopsis using open chromatin signature.](http://zhangtaolab.org/pdf/2015/Plant_Cell_2015.pdf) ***The Plant Cell*** 2015, 27(9):2415-2426.

## 2014

* Zhang T, Li GR, Yang ZJ, Nevo E. Adaptive evolution of duplicated hsp17 genes in wild barley from microclimatically divergent sites of Israel. ***Genetics and Molecular Research*** 2014, 13(1):1220-1232.

* Zhang WL, Zhang T, Wu YF, Jiang JM. Open Chromatin in Plant Genomes. ***Cytogenetic and Genome Research*** 2014(1-3), 143:18-27.

* Yang LM†, Koo, DH†, Li D, Zhang T, Jiang JM, Luan FS, Renner SS, Henaff E, Sanseverino W, Garcia-Mas J, Casacuberta J, Senalik DA, Simon PW, Chen JF, and Weng YQ. Next-generation sequencing, FISH mapping and synteny-based modeling reveal mechanisms of decreasing dysploidy in *Cucumis*. ***The Plant Journal*** 2014, 77(1):16-30.

## 2013

* Zhang T†, Talbert PB†, Zhang WL†, Wu YF, Yang ZJ, Henikoff JG, Henikoff S, Jiang JM. [The CentO satellite confers translational and rotational phasing on cenH3 nucleosomes in rice centromeres.](http://zhangtaolab.org/pdf/2013/PNAS_2013.pdf) ***Proceedings of the National Academy of Sciences of the United States of America*** 2013, 110(50):E4875–E4883.

  * Commentary: Heslop-Harrison, J.S. and Schwarzacher, T. [Nucleosomes and centromeric DNA packaging.](http://zhangtaolab.org/pdf/2013/PNAS_2013_comm.pdf) ***Proceedings of the National Academy of Sciences of the United States of America*** 2013, 110(50):19974-19975.

* Iovene M, Zhang T, Lou Q, Buell CR, Jiang JM. Copy number variation in potato - an asexually propagated autotetraploid species. ***The Plant Journal*** 2013, 75(1):80-89.

* Wei W, Zhang T, Lin D, Yang ZJ, Guo FB. Transcriptional abundance is not the single force driving the evolution of bacterial proteins. ***BMC Evolutionary Biology*** 2013, 13(1):162.

* Yang ZJ, Zhang T, Lang T, Li G, Chen G, Nevo E. Transcriptome Comparative Profiling of Barley *eibi1* Mutant Reveals Pleiotropic Effects of *HvABCG31* Gene on Cuticle Biogenesis and Stress Responsive Pathways. ***International Journal of Molecular Sciences*** 2013, 14(10):20478-20491.

* Dong ZB, Jiang C, Chen X, Zhang T, Ding L, Song W, Luo H, Lai J, Chen H, Liu R, Jin WW. Maize LAZY1 Mediates Shoot Gravitropism and Inflorescence Development through Regulating Auxin Transport, Auxin Signaling, and Light Response. ***Plant Physiology*** 2013, 163(3):1306-1322.

## 2012

* Zhang WL†, Zhang T†, Wu YF†, Jiang JM. [Genome-Wide Identification of Regulatory DNA Elements and Protein-Binding Footprints Using Signatures of Open Chromatin in *Arabidopsis*.](http://zhangtaolab.org/pdf/2012/Plant_Cell_2012.pdf) ***The Plant Cell*** 2012, 24(7):2719-2731.

## 2011

* Yang ZJ, Zhang T, Li GR, Nevo E. Adaptive microclimatic evolution of the dehydrin 6 gene in wild barley at “Evolution Canyon”, Israel. ***Genetica*** 2011, 139(11-12):1429-1438.

* Wang XH, Zhang T, Wen ZN, Xiao H, Yang ZJ, Chen GX, Zhao X. The chromosome number, karyotype and genome size of the desert plant diploid *Reaumuria soongorica* (Pall.) Maxim. ***Plant Cell Reports*** 2011, 30(6):955-964.

* Tang ZX, Fu SL, Ren ZL, Zhang T, Zou YT, Yang ZJ, Li GR, Zhou JP, Zhang HQ, Yan BJ, Zhang HY, Tan FQ. Diversity and evolution of four dispersed repetitive DNA sequences in the genus *Secale*. ***Genome*** 2011, 54(4):285-300.

## 2010

* Li GR, Zhang T, Wei P, Jia J, Yang ZJ. Sequence analysis of alpha-gliadin genes from *Aegilops tauschii* native to China. ***Asian Journal of Agricultural Sciences*** 2010, 2(4):128-135.

* Li GR, Zhang T, Ban Y, Yang ZJ. Molecular characterization and evolutionary analysis of alpha-gliadin genes from *Eremopyrum bonaepartis* (Triticeae). ***Journal of Agricultural Science*** (Toronto) 2010, 2(4):30-36.

## 2009

* Yang ZJ, Zhang T, Bolshoy A, Beharav A, Nevo E. Adaptive microclimatic structural and expressional dehydrin 1 evolution in wild barley, *Hordeum spontaneum*, at ‘Evolution Canyon’, Mount Carmel, Israel. ***Molecular Ecology*** 2009, 18(9):2063-2075.

* Yang ZJ, Li GR, Jia JQ, Zeng X, Lei MP, Zeng ZX, Zhang T, Ren ZL. Molecular cytogenetic characterization of wheat–*Secale africanum* amphiploids and derived introgression lines with stripe rust resistance. ***Euphytica*** 2009, 167(2):197-202.

* Li GR, Liu C, Zeng ZX, Jia J-Q, Zhang T, Zhou JP, Ren ZL, Yang ZJ. Identification of α-gliadin genes in *Dasypyrum* in relation to evolution and breeding. ***Euphytica*** 2009, 165(1):155-163.

## 2006

* Hu GK, Feng H, Zhang T, Yan Y, Wu B, Jiang Q, Wu JM, Zhang YZ. Molecular cloning of cDNAs for 14-3-3 and its protein interactions in a white-rot fungus *Phanerochaete chrysosporium*. ***Annals of Microbiology*** 2006, 56(3):191-196.
